metacat-connector-hive/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+dependencies {
+    compile project(':metacat-common-server')
+    compile project(':metacat-common')
+    compile "org.apache.hive:hive-metastore:1.2.1"
```

Comment:
Move versions to gradle.properties for consistency

We should switch to dependency management system soon